FAQ.md

```diff
@@ -189,16 +187,15 @@ Due to a bug in Android this is not possible.
 See [here](https://github.com/M66B/NetGuard/issues/42) for more information.
 
 <a name="FAQ24"></a>
-**(24) Can you remove the notification from the status bar?**
+**(24) Can you remove the notification from the notification screen?**
```

Comment:
The top system bar is called the status bar
The second application bar is called the action bar

Don't blame me ...